### PR TITLE
refactor: clean up

### PR DESCRIPTION
**Pull request by anton-karlovskiy**

### CODE REVIEW

File: src/components/TestnetBanner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+import { useTranslation } from 'react-i18next';
+
+function TestnetBanner() {
+  const { t } = useTranslation();
+
+  return (
+    <div
+      style={{ margin: 16 }}
```
Comment (nud3l) [inline src/components/TestnetBanner/index.tsx:9]: Can we please make the note the same width as the underlying page (70% at fullscreen)?

Your change:

image

Existing notice:
image

**Review by nud3l — CHANGES_REQUESTED** [review]

File: src/components/TestnetBanner/index.tsx (hunk 1 above)
Comment (anton-karlovskiy) [reply src/components/TestnetBanner/index.tsx:9]: I set it to 60% as 70% was also wide.
Please review it.

**Review by nud3l — DISMISSED** [review]

**Review by nud3l — CHANGES_REQUESTED** [review]
LGTM, except for one minor thing. When you navigate from a page to the landing page there is this split second incorrect rendering:


https://user-images.githubusercontent.com/14966470/109871090-27463b80-7cae-11eb-8862-cf4faabebeec.mp4


https://user-images.githubusercontent.com/14966470/109871098-29a89580-7cae-11eb-97ef-b7520f738c2c.mp4

**Comment by anton-karlovskiy** [comment]
@nud3l 
Yes, I noticed this UX bug.
I think there's something conflicting regarding code-splitting + transition + routing.
For now, I blocked that transition and continue to fix it progressively. (TODO for now)

**Review by nud3l — APPROVED** [review]

**Pull request merged**